ui/config-overrides.js

```diff
@@ -0,0 +1,39 @@
+const webpack = require('webpack');
+
+module.exports = {
+  webpack: function(config, env) {
+    let environmentFilePath = 'environment.localtest.ts'
+    if (process.env.REACT_APP_ENVIRONMENT === 'local' ) {
+      environmentFilePath = 'environment.local.ts'
```

Comment:
nit: we can try something like `environmentFilePath = 'environment.{process.env.REACT_APP_ENVIRONMENT}.ts'`